jetstream/core/config_lib.py

```diff
@@ -85,11 +84,16 @@ class InterleavedCPUTestServer(ServerConfig):
 
 
 def slice_to_num_chips(s: str) -> int:
-  """Converts a TPU spec like v5e=4x2 to the number of chips, 8."""
-  # Account for the case where it is written 'v5e:4x2'.
-  delim = "=" if "=" in s else ":"
-  i = math.prod([int(c) for c in s.split(delim)[1].split("x")])
-  return i
+  """Converts a TPU spec like v5e-8 or v5e=8 to the number of chips, 8."""
+  # Account for the case where it is written 'tpu=8' for compatibility.
+  delim = "-" if "-" in s else "="
+  # TODO: Support more accelerator type check.
+  accelerator_type, num_devices = s.split(delim)
+  return (
+      int(num_devices)
+      if accelerator_type != "v4"
```

Comment:
Is there any reason to only return half of chips?

Reply:
V4-8 has 8 tensor cores but 4 chips (2 cores per chip)